src/components/Layouts/MarsRoverPage/index.tsx

```diff
@@ -1,10 +1,13 @@
 import React from "react";
 import "./styles.scss";
+import { APOD } from "../../APOD/apod";
 
 const MarsRoverPage: React.FC = () => {
   return (
     <div className="gutter">
-      <div className="mars-rover-page layout">hi</div>
+      <div className="mars-rover-page layout">
```

Comment:
sorry this was me, would you mind removing while you do the other markups please! thank you!

Reply:
Done!